queries.ghactivity.php

```diff
@@ -494,4 +494,82 @@ public static function get_monitored_repos( $fields = 'id=>name' ) {
 
 		return $repos_to_monitor;
 	}
+
+	public static function fetch_project_stats( $org_name, $project_name, $range = null ) {
+		$slug = $org_name . '#' . $project_name;
+		$args = array(
+			'post_type'      => 'gh_query_record',
+			'post_status'    => 'publish',
+			'posts_per_page' => -1,
+			'orderby'        => 'date',
+			'order'          => 'ASC',
+			'tax_query'      => array(
+				array(
+					'taxonomy' => 'ghactivity_query_project_slug',
+					'field'    => 'name',
+					'terms'    => $slug,
+				),
+			),
+		);
+
+		// FIXME: Add caching
+		$posts = get_posts( $args );
+
+		function get_post_content( $post ) {
+			return array(
+				'post_content' => $post->post_content,
+				'post_date'    => strtotime( $post->post_date ),
+				'columns'      => json_decode( get_post_meta( $post->ID, 'recorded_columns', true ) ),
+				'project_url'  => get_post_meta( $post->ID, 'project_url', true ),
```

Comment:
It may be good to escape the data here, and for the rest of the info in the array.